Conda doc patch (#30)
moj-analytical-services/user-guidance: source/documentation/appendix-docs/conda.md

#### Examples

The following tables show conda commands and their base R analogues.

> - Installing a package:

Reviewer: I think it would better without bullet points, especially as the tables will not share the same indentation.
```suggestion
Installing a package:
```

<div style="height:0px;font-size:0px;">&nbsp;</div>
| `install.packages` (in R-Console) | `conda install` (in Terminal) |
| --------------------------------- | ----------------------------- |
| `install.packages('Rcpp')`        | `conda install r-Rcpp`        |
<div style="height:0px;font-size:0px;">&nbsp;</div>

![](images/conda/conda_install_rcpp.gif)

> - Installing a specific version of a package

Reviewer: ```suggestion
Installing a specific version of a package:
```

#### Packrat

Packrat is the most well-known package management tool for R. There's more information about it here: <https://rstudio.github.io/packrat/>

It has some significant downsides. It can be quite temperamental, and difficult to debug when things go wrong - in the earlier days of the Analytical Platform, the majority of support issues related to getting Packrat working. 

> Furthermore, the Platform version of RStudio runs on a Linux virtual machine, and CRAN mirrors do not provide Linux compiled binaries for packages. This means that packages need to be compiled on the Analytical Platform every time they're installed, which can take a long time. This means a long wait when doing `install.packages` both in an RStudio session, and when running a Docker build for a `shiny` application.

Reviewer: ```suggestion
Furthermore, the Analytical Platform version of RStudio runs on a Linux virtual machine and CRAN mirrors do not provide Linux compiled binaries for packages. This means that packages need to be compiled on the Analytical Platform every time they're installed, which can take a long time. This also means a long wait when doing `install.packages`, both in an RStudio session, and when running a Docker build for an RShiny app.
```
